app/controllers/post_controller.rb

```diff
@@ -1,9 +1,13 @@
 class PostsController < ApplicationController
   def index
-    @posts = Post.all
+    @user = User.find(params[:user_id])
```

Comment:
**You did a great job! The implementation looks astonishing from the user's perspective.** 

- [ ] However, I'm facing an issue with the Posts view. Your posts are not showing up. The problem is because you didn't make your controller name plural. It should be 'Posts' instead of 'Post'. By changing your controller's name, your problem will be solved :)